components/core/DownloadDaemon.py

```diff
@@ -7,6 +7,7 @@
 from EMail import send_mail
 from DiskMan import *
 from ConfReader import get_conf_reader
+from fileminio import *
```

Comment:
Import only required things from fileminio as it will give more readability to new contributor if they want to know from where this function is inherited